tests/test_link_io_structure.py

```diff
@@ -95,7 +95,32 @@ def test_overwrite_symlink_stdout(tmpdir, capsys):
         os.symlink("2023.06", "2023")
         create_symlinks(links_dict)
         captured = capsys.readouterr()
-        assert "Existing symlink 2023 moved from 2023.06 to 2023.07\n" == captured.out
+        print(captured.out)
```

Comment:
leftover print?

Reply:
Yes, is it possible to configure pylint to catch leftover print statements in tests?

Reply:
Don't think so.